src/main/java/com/djrapitops/extension/CoreProtectExtension.java

```diff
@@ -47,6 +47,7 @@ of this software and associated documentation files(the "Software"), to deal
     private static final int ACTION_REMOVED = 0;
     private static final int ACTION_PLACED = 1;
     private static final int ACTION_INTERACT = 2;
+    private static final int ACTION_CHAT = 3; // Not sure of the value, need trials.
```

Comment:
Did you manage to test this? :)

Reply:
Not really, I don't have the knowledge to build the plugin :/ Sorry, that's why I'm asking you xD

Reply:
**EDIT** : I found, I'll let you know :)